lisa/target.py

```diff
@@ -115,7 +115,7 @@ class TargetConf(SimpleMultiSrcConf, HideExekallID):
         KeyDesc('kind', 'Target kind. Can be "linux" (ssh) or "android" (adb)', [str]),
 
         KeyDesc('host', 'Hostname or IP address of the host', [str, None]),
-        KeyDesc('username', 'SSH username', [str, None]),
+        KeyDesc('username', 'SSH username. On ADB connections, "root" username will root adb upon target connection', [str, None]),
```

Comment:
I'm really not keen on the double meaning of this key. Could we have an ADB specific key (like `device`) instead?

Reply:
It's not actually double meaning, since requesting "root" will actually log you as root on the device, just like for SSH. If possible, I'd rather avoid proliferation of mutually-exclusive or context specific keys at the top-level, especially if their meaning 100% overlap. The only real reason `device` exists is that `host` can also be used for ADB, so we needed another one.